CHANGELOG.md

```diff
@@ -38,11 +38,19 @@ Format:
 ### Security
 - 
 -->
-<!-- ## [Unreleased] -->
+## [Unreleased]
+### Changed
+- Ensure unique folder names for random input screenshots in case developer uses multiple random steps in one Scenario.
+- Ensure random input detects unexpected interview errors.
+
+### Fixed
+- Fix failure when signature page is the first screen.
```

Comment:
```suggestion
- Fix failure when signature page comes right before an error screen.
```

Reply:
Actually, the test itself failed when the signature page was the first page. `scope.getLoadData()` didn't recognize it as loading properly.

https://github.com/SuffolkLITLab/ALKiln/pull/591/files#diff-4b632aedc4c91285de358dd7cbcc91a52724745c5c759b81b3a6d80e8d6d7abcR622